packages/api/test/post-build-test.js

```diff
@@ -261,6 +262,58 @@ describe('Posting Builds', () => {
     assert.strictEqual(result.data().environment.ref, 'master');
   });
 
+  describe('xunit endpoint', async () => {
+    let stubs = [];
+
+    afterEach(() => {
+      /** Cleanup **/
+      stubs.forEach(stubbed => {
```

Comment:
the pattern I usually use here is to create a sandbox, and then in the `afterEach` call `sandbox.restore()`.  That would help you skip keeping track of all your used stubs:

```js
const sandbox = sinon.createSandbox();
..
const stub1 = sandbox.stub(obj, 'method')....
const stub2 = sandbox.stub(obj, 'method')....

# in your afterEach
sandbox.restore();
```

Reply:
Oh yeah that's much more elegant, thank you!